### ISSUE TITLE
[DevOps] Staging implementation for `CHIP` IaC Terraform rewrite

### ISSUE THREAD

**Issue by ndouglas** [department-of-veterans-affairs/va.gov-team#79801]
## User Story
As an engineer, I want to rewrite the Staging environment for the `CHIP` IaC in Terraform to improve reliability and maintainability.

Followup to #79798. Outcomes from that ticket will be used to populate and refine this ticket.

## Tasks
- [ ] [DevOps] **_insert DevOps work_**
- [ ] [DevOps] Brief postmortem of Staging environment work to inform Prod followup ticket (#79802).
- [ ] [DevOps] Review production template to see what further things need discovery.

## Acceptance Criteria

- [ ] Postmortem in thread of things that were unanticipated.
- [ ] Production implementation ticket (#79802) is updated with information outcomes from this ticket.

**Comment by ndouglas** [comment]
Closed as Eric did this work in parallel with the work on #79798.
